pythonfile.py

```diff
@@ -17,10 +17,17 @@ def something(number):
 
 print(something(0))
 
+
+print("LOTS OF PRINTS!!")
+print("LoTs")
+print("oF")
+print("prINtS")
+
```

Comment:
## Good looking merge

I like how we kept 'lots of prints' and 'pascals triangle'

*Nice EM!*